From 2396e32dafe91e1fc5474bd0937ea4954ba990ba Mon Sep 17 00:00:00 2001
From: Richard Lowe <richlowe@richlowe.net>
Date: Tue, 16 Jul 2019 06:47:17 +0000
Subject: [PATCH] sb-concurrency: increase the test-frlock timeout radically

I have no idea whether this is the right thing to do, but it seems like the
test itself is running fine just much more slowly than anticipated.
---
 contrib/sb-concurrency/tests/test-frlock.lisp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/sb-concurrency/tests/test-frlock.lisp b/contrib/sb-concurrency/tests/test-frlock.lisp
index 6507e7b49..65655960e 100644
--- a/contrib/sb-concurrency/tests/test-frlock.lisp
+++ b/contrib/sb-concurrency/tests/test-frlock.lisp
@@ -88,7 +88,7 @@
 #+sb-thread
 (deftest* (frlock.1)
     (handler-case
-        (sb-ext:with-timeout 20
+        (sb-ext:with-timeout 110
           (test-frlocks #+win32 :outer-write-pause #+win32 t ))
       (sb-ext:timeout (c)
         (error "~A" c)))
-- 
2.21.0

